src/MovieCard.jsx

```diff
@@ -42,33 +26,16 @@ const MovieCard = (props) => {
         setIsLiked(!isLiked);
     }
 
-    const toggleWatched = () => {
-        setIsWatched(!isWatched);
-    }
-
-    // const addLikedMovie = (movieTitle, movieImage) => {
-    //     const newLikedList = [...likedMovies, [movieTitle, movieImage]];
-    //     setLikedMovies(newLikedList);
-    //     // console.log(likedMovies);
-    // }
-
-    const addWatchedMovie = (movieTitle, movieImage) => {
-        if (!isChecked) {
-            const newWatchedList = [...watchedMovies, {title: movieTitle, image: movieImage}];
-            setWatchedMovies(newWatchedList);
-        }else {
-            const newWatchedList = watchedMovies.filter(movie => movie.title !== movieTitle);
-            setWatchedMovies(newWatchedList);
-        }
-        // console.log(isChecked);
-        // console.log(watchedMovies);
-    }
-
-
     const increaseLikes = () => {
-        const heart = document.getElementById(props.id).querySelector('#like-count');
-        setLikeCount(likeCount + 1);
-        heart.innerText = `❤️ ${likeCount + 1}`;
+        if (likeCount === 0) {
+            const heart = document.getElementById(props.id).querySelector('#like-count');
+            setLikeCount(likeCount + 1);
+            heart.innerText = `❤️`;
+        } else if (likeCount >= 1) {
+            const heart = document.getElementById(props.id).querySelector('#like-count');
+            setLikeCount(likeCount - 1);
+            heart.innerText = `♡`;
```

Comment:
The function is called "increaseLikes", but the likeCount is decreased here. Do you want to update the logic,  or rename the function and use bool state to indicate whether movie is liked or not?